seahub/templates/js/templates.html

```diff
@@ -36,7 +36,11 @@
       <img src="{{ MEDIA_URL }}img/sync-folder-20.png" title="{% trans "Read-Write" %}" alt="" />
       <% } %>
     </td>
+    <% if (name) { %>
     <td><a href="#my-libs/lib/<%= id %>"><%- name %></a></td>
+    <% } else { %>
+    <td>(Broken)</td>
```

Comment:
Why does it only says (Broken) without any information? Wouldn't it still be helpful to put the id in? Maybe with a small info icon next to it which says something like `please contact your administrator providing the library id to fix this library`?